cyclonedx_py/parser/environment.py

```diff
@@ -43,19 +44,25 @@
 else:
     from importlib_metadata import metadata, PackageMetadata as _MetadataReturn
 
-from cyclonedx.model import LicenseChoice
+from cyclonedx.model import License, LicenseChoice
 from cyclonedx.model.component import Component
 from cyclonedx.parser import BaseParser
 
 
+@enum.unique
+class LICENSE_OUTPUT_FORMAT(enum.Enum):
```

Comment:
this is an enum, not a constant. user proper casing ala pep8 - <https://peps.python.org/pep-0008/#class-names>

is there a pep that encourages all-upper-case for enums?